### ISSUE TITLE
Error:  could not find function "loo_flock_onefit"

### ISSUE THREAD

**Issue by abfleishman** [jsocolar/flocker#44]
This makes no sense to me.  I see that the [roxygen2 does not export](https://github.com/jsocolar/flocker/blob/6ea4dbc35d8d4602b589e58ae9240a6f0f5ba693/R/loo.R#L31-L50)  `loo_flock_onefit` but because it is being used internally, it should not matter.  

```
mod_loo<-loo_compare_flocker(list(mod_null, mod, mod1))
```

> Error in loo_flock_onefit(x[[i]]) : 
>   could not find function "loo_flock_onefit"

```
mod_loo<-loo_flocker(mod_null)
```

> Error in loo_flock_onefit(x) : could not find function "loo_flock_onefit"

If I use the `:::` notation I can print out the function so it clearly exists!
```
flocker:::loo_flocker_onefit
```

> function (x) 
> {
>     type <- type_flocker_fit(x)
>     if (type == "C") {
>     }
>     else if (type == "V") {
>         chain_id <- rep(c(1:dim(x$fit)[2]), each = dim(x$fit)[1])
>         ll <- log_lik_V(x)
>         out <- loo::loo(ll, r_eff = loo::relative_eff(ll, chain_id = chain_id))
>     }
>     return(out)
> }
> <bytecode: 0x000001fa0d95aa18>
> <environment: namespace:flocker>

**Comment by abfleishman** [comment]
Oh!  There is a name issue! It is calling `loo_flock_onefit` and it needs to be changed to `loo_flocker_onefit`!

**Comment by jsocolar** [comment]
Closed via #45
